pymchelper/readers/fluka.py

```diff
@@ -340,36 +342,41 @@ def get_unit_for_scoring(cls, scoring: str) -> str:
         :param scoring: scoring name
         :return: tuple of scoring and unit
         """
+        identity = 1.0
+
         if scoring in cls._deposition_scorings:
             if scoring == 'DPA-SCO':
-                return '/g'
+                return '/g', identity
             if 'DOSE' in scoring:
-                return 'GeV/g '
-            return 'GeV'
+                # Doses are expressed in GeV/g per unit primary weight.
+                # To obtain dose in Gy, we multiply it by 1.602176462E-7.
+                return 'Gy', 1.602176462E-7
```

Comment:
As for DOSE from Fluka I'd stay with default unit of DOSE SHIELD-HIT12A scorer (MeV/g?).
SHIELD-HIT12A has a dedicated scorer to score dose in Gy, called DOSEGY.